src/web/mjs/connectors/Delitoon.mjs

```diff
@@ -50,12 +50,12 @@ export default class Delitoon extends Connector {
         return data.filter(element => element.querySelector('div.number'))
             .map(element => {
                 let title = element.querySelector('div.container-infos div.bloc-infos div.number').textContent.trim();
-                let link = element.querySelector('li a');
+                let link = element.closest('a');
```

Comment:
The search direction has been changed, but the chapter query is still the same, was this intentional?
`querySelectors()` searches the DOM tree downwards (children)
`closest()` searches the DOM tree upwards (parents)

Reply:
Already reverted with successive commit